src/_pytest/outcomes.py

```diff
@@ -159,10 +159,15 @@ def importorskip(modname, minversion=None, reason=None):
         # import but without a __init__.py file
         warnings.simplefilter("ignore")
         try:
-            __import__(modname)
+            mod = __import__(modname)
         except ImportError:
             # Do not raise chained exception here(#1485)
             should_skip = True
+        else:
+            if mod.__file__ is None:
```

Comment:
I'm 👎 on this -- with python2.7 disappearing I'll be deleting as many `__init__.py` files as I can and I suspect others will as well

also builtin modules won't have `__file__`, and neither will things like `six.moves`

Reply:
Is there a better way to detect this (the package not being deinstalled completely)?

Reply:
I don't think so unfortunately -- there's also (rare) cases of people using pyc-only distributions which i think this would also false-positive on as well?